core/iwasm/doc/wasm_exports.MD

```diff
@@ -0,0 +1,22 @@
+# Wasm exports
+The internal data structure for Wasm exports:  
+![](./images/wasm_exports.svg)  
+
+## Setup exports for Module 
+The array data structure pointed by `WASMModule::exports` is setup during loading a module. Basically the runtime will load the exports sections from the module file content, and construct an array of C struct `WASMExport`.   
+
+A `WASMExport` item contains three elements that map the Wasm file exports section structure:
+- name: the name shown to external
+- kind: total 4 export types: function, globals, memory, table
+- index: As all the 4 export types are organized in array, this refers to index in target export type 
+
+## Function exports
+### use function exports
+function exports are often used in two situations:  
+    1. **call by host**: runtime API `wasm_runtime_lookup_function` will walk through the array of `WASMModuleInstance::export_functions` and compare the exported name with given target symbol name in the function parameter. If any array item matches the name, then return the value of field `function` which points to associated function instance (WASMFunctionInstance)  
```

Comment:
Had better?
If any array item matches the name, then return the value of field
->
If any array item matches the name, it then returns the value of field

Reply:
ok, I will update it according to your comments